### ISSUE TITLE
scan in docker failed - missing log file

### ISSUE THREAD

**Issue by iammyr** [EFForg/badger-sett#32]
When running `./runscan.sh` I get the following error:

```
Running scan in Docker...
Ctrl-C to break.
Traceback (most recent call last):
  File "./crawler.py", line 620, in <module>
    crawler = Crawler(**vars(ap.parse_args()))
  File "./crawler.py", line 186, in __init__
    fh = logging.FileHandler(os.path.join(out_path, 'log.txt'))
  File "/usr/lib/python3.5/logging/__init__.py", line 1008, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.5/logging/__init__.py", line 1037, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
PermissionError: [Errno 13] Permission denied: '/home/<local user>/out/log.txt'
mv: cannot stat '/home/<local user>/<local path to repo>/docker-out/log.txt': No such file or directory
Scan failed. See log.txt for details.
```

**Comment by ghostwords** [comment]
Is this fixed by #48?

**Comment by bcyphers** [comment]
This should have been fixed in #48, closing.
